### PR TITLE
{data}[foss/2021a] Cartopy v0.20.0, Fiona v1.8.20, pyproj v3.1.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/4b503e060b7cb452f4892288f19c6570 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
jsfc061 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/8cd4bec0ae12194334f267e555dc4381 for a full test report.

**Review by SebastianAchilles — COMMENTED** [review]
I think it makes sense to keep the `sanity_check_commands`.

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }),
+]
+
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/fio'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
```
[the remark this answers is not in the document]
Comment (maxim-masterov) [reply easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2021a.eb:44]: done

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/f5978bc88676331ae3b1e76b663ef0da for a full test report.

**Comment by SebastianAchilles** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**